assets/js/gql/FileSetFields.frag.gql

```diff
@@ -0,0 +1,11 @@
+fragment FileSetFields on FileSet {
+  id
+  accessionNumber
+  role
+  rank
+  position
+  metadata {
+    description
+    digests
+  }
+}
```

Comment:
Are we going to keep using these `gql` files?  The front end (for now) is not using them...just wondering if housing in the `/assets/` folder is best place?

Reply:
We can move them. I'll open an issue to move them out of the front end.

Reply:
Maybe to `/test/gql/` or `/test/fixtures/gql/`?

Reply:
@mbklein - can you add your thoughts to the issue?
https://github.com/nulib/repodev_planning_and_docs/issues/435